src/openpower.md

```diff
@@ -0,0 +1,28 @@
+# OpenPOWER boot chain
+
+```mermaid
+flowchart LR
+    skiroot(["skiroot (petitboot)"])
+    SBEs-->hostboot-->skiboot-->skiroot-->OS
+```
+
+- [Self-boot engines](https://github.com/open-power/sbe) (SBEs) are split
+  between an on-chip ROM and an external EEPROM
+- [hostboot](https://github.com/open-power/hostboot) is a C++ boot loader that
+  does DRAM initialization provides runtime services to skiboot or phyp, a
+  hypervisor
+- [skiboot](https://github.com/open-power/skiboot) is a C boot loader and
+  runtime firmware for OpenPOWER that loads skiroot.
+
+skiroot is a term used to describe the LinuxBoot implementation for OpenPOWER.
+There is no skiroot repository nor a package with that name. It is only used in
+the kernel config, `skiroot_defconfig`.[^1]
```

Comment:
Probably worth mentioning the `op-build` infrastructure instead, which constructs the petitboot environment. Maybe something like:

```
The petitboot stage is a small kernel and initramfs-based root filesystem,
built using the `op-build` infrastructure, to provide boot-device discovery,
and a bootloader UI.
```

Reply:
Absolutely, I just didn't get that far when Daniel and I were digging into all this last night. It would also be nice to explain where things are in mainline Buildroot, the Buildroot fork, the overlay (I assume this was created before br2_external), etc. Reference what the userspace is made of like Busybox/SysVinit, etc. Also dig into the kernel defconfig to understand what options were selected and why. And then what are the features of petitboot besides just calling kexec. I think I saw some screenshots a TUI at some point.